roles/reproducer/tasks/devscripts_layout.yml

```diff
@@ -1,6 +1,337 @@
+      ansible.builtin.include_role:
+        name: "virtualbmc"
+        tasks_from: "list_hosts.yml"
+
+    - name: Generate baremetal content fact
+      vars:
+        _hypervisor_ip4: >-
+          {{ hostvars[inventory_hostname].ansible_default_ipv4.address }}
+        _vm_name: "cifmw-{{ item }}"
+        _mac: >-
+          {{
+            cifmw_libvirt_manager_mac_map[item | replace('_master', '')] |
+            selectattr('network', 'equalto', cifmw_reproducer_provision_net) |
+            map(attribute='mac') | first
+          }}
+        _ipmi_port: >-
+          {{
+            cifmw_virtualbmc_known_hosts |
+            selectattr('Domain name', 'match', _vm_name) |
+            map(attribute='Port') | first
+          }}
+        _name: >-
+          {{
+            item |
+            regex_replace('^ocp_master-([0-9]+)$', 'openshift-master-\1') |
+            regex_replace('^ocp_worker-([0-9]+)$', 'openshift-worker-\1')
+          }}
+        _node:
+          name: "{{ _name }}"
+          driver: "ipmi"
+          resource_class: "baremetal"
+          driver_info:
+            username: "admin"
+            password: "password"
+            address: "ipmi://{{ _hypervisor_ip4 }}:{{ _ipmi_port }}"
+            deploy_kernel: >-
+              http://{{ _pxe_host }}/images/ironic-python-agent.kernel
+            deploy_ramdisk: >-
+              http://{{ _pxe_host }}/images/ironic-python-agent.initramfs
+            disable_certificate_verification: false
+          ports:
+            - address: "{{ _mac }}"
+              pxe_enabled: true
+          properties:
+            local_gb: "50"
+            cpu_arch: "x86_64"
+            boot_mode: "UEFI"
+      ansible.builtin.set_fact:
+        _ironic_bm_data: >-
+          {{
+            (_ironic_bm_data | default([])) + [_node]
+          }}
+      loop: "{{ _cifmw_libvirt_manager_ocp_cluster }}"
+      loop_control:
+        index_var: vm_id
+        label: "{{ item }}"
+
+    - name: Output baremetal content in file
+      vars:
+        _content:
+          nodes: "{{ _ironic_bm_data }}"
+      ansible.builtin.copy:
+        content: "{{ _content | to_nice_json }}"
+        dest: "{{ _bm_ironic_file }}"
+        mode: "0644"
+
+    - name: Group devscripts last actions before deployment
+      vars:
+        _pub_net_data: >-
+          {{
+            _libvirt_manager_networking.networks[cifmw_libvirt_manager_pub_net]
+          }}
+        _bootstrap_mac: "52:54:ab:83:31:87"
+      block:
+        - name: Generate patched overrides for devscripts
+          vars:
+            _num_workers: >-
+              {{
+                _cifmw_libvirt_manager_layout.vms.ocp_worker.amount | default(0)
+              }}
+            _num_masters: >-
+              {{
+                _cifmw_libvirt_manager_layout.vms.ocp.amount | default(3)
+              }}
+          ansible.builtin.set_fact:
+            cifmw_devscripts_config_overrides_patch_02_baremetal:
```

Comment:
I think somewhere along the way we lost something setting the MAC address for the external (ocpbm) interfaces to `_bootstrap_mac` here?

https://github.com/openshift-metal3/dev-scripts/blob/master/config_example.sh#L392

```
[zuul@host26 ci-framework-data]$ virsh domiflist ocp-7fhmz-bootstrap
 Interface   Type     Source   Model    MAC
-----------------------------------------------------------
 vnet0       bridge   ocpbm    virtio   52:54:00:66:56:37
 vnet1       bridge   ocppr    virtio   52:54:00:a6:be:1e
```

```
[zuul@host26 ci-framework-data]$ grep ocp-boostrap /etc/cifmw-dnsmasq.d/ocpbm-hosts.conf 
dhcp-host=52:54:ab:83:31:87,[fd00:1:2::4],ocp-boostrap
```

The MACs don't match up, so the bootstrap nodes boots into no networking and fails `Unable to pull OpenShift release image`.

Reply:
right, I lost it during some content moving -.-. It's back in place with latest push - sorry for that one!